src/components/application_manager/test/commands/mobile/on_driver_distraction_notification_test.cc

```diff
@@ -0,0 +1,112 @@
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * Redistributions of source code must retain the above copyright notice, this
+ * list of conditions and the following disclaimer.
+ *
+ * Redistributions in binary form must reproduce the above copyright notice,
+ * this list of conditions and the following
+ * disclaimer in the documentation and/or other materials provided with the
+ * distribution.
+ *
+ * Neither the name of the Ford Motor Company nor the names of its contributors
+ * may be used to endorse or promote products derived from this software
+ * without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#include <stdint.h>
+#include <string>
+
+#include "gtest/gtest.h"
+#include "utils/shared_ptr.h"
+#include "smart_objects/smart_object.h"
+#include "application_manager/smart_object_keys.h"
+#include "application_manager/commands/commands_test.h"
+#include "application_manager/commands/command_impl.h"
+#include "application_manager/mock_message_helper.h"
+#include "mobile/on_driver_distraction_notification.h"
+
+namespace test {
+namespace components {
+namespace commands_test {
+namespace mobile_commands_test {
+
+namespace am = ::application_manager;
+
+using ::testing::_;
+using ::testing::Return;
+using ::testing::Mock;
+using am::commands::CommandImpl;
+using am::MockMessageHelper;
+using am::commands::MessageSharedPtr;
+using am::commands::mobile::OnDriverDistractionNotification;
+
+typedef ::utils::SharedPtr<OnDriverDistractionNotification> CommandPtr;
+
+class MobileOnDriverDistractionNotificationTest
+    : public CommandsTest<CommandsTestMocks::kNotNice> {
+ public:
+  MobileOnDriverDistractionNotificationTest()
+      : mock_message_helper_(*MockMessageHelper::message_helper_mock()) {}
+
+  void TearDown() OVERRIDE {
+    Mock::VerifyAndClearExpectations(&mock_message_helper_);
+  }
+
+  void SetSendNotificationExpectations(MessageSharedPtr& msg) {
+    Mock::VerifyAndClearExpectations(&mock_message_helper_);
+    EXPECT_CALL(mock_message_helper_, PrintSmartObject(_))
```

Comment:
@AGaliuzov, I don't think that is good idea. This one: 
- Have no effect on command execution at all;
- It may provides additional visual noise,
  which may only confuse, rather then help;
- And if someone will delete it form command realization
  it will cause test failure, obviously, for no necessary reason.